views/includes/global-header-next.njk

```diff
@@ -54,6 +66,28 @@
     </ul>
 {% endmacro %}
 
+{% macro globalSearch() %}
+    <form class="global-search-next" role="search" action="/search">
+        <input type="hidden" name="lang" value="en-GB">
```

Comment:
Just looking at this... wonder if we still need these parameters now the search goes to Google?